cmd/config/internal/commands/cmdcreatesetter_test.go

```diff
@@ -463,6 +463,191 @@ spec:
   hub: my-hub # {"type":"","x-kustomize":{"setter":{"name":"hubsetter","value":"my-hub"}}}
  `,
 		},
+		{
```

Comment:
Can we have a test for unknown fields being dropped?

Reply:
Done